saleor/app/models.py

```diff
@@ -124,6 +124,17 @@ def has_perm(self, perm: BasePermissionEnum | str) -> bool:
         return perm_value in self.get_permissions()
 
 
+class AppWebhookMutex(models.Model):
+    app = models.OneToOneField(
+        App,
+        on_delete=models.CASCADE,
+        related_name="webhook_mutexes",
+        verbose_name="App",
+    )
+    acquired_at = models.DateTimeField(null=True)
+    uuid = models.UUIDField(default=uuid4, unique=True)
```

Comment:
I know it's still draft but just as a side note - it might be good to set this as primary key. In migration we have extra ID column which I don't think we're gonna use at all.

Reply:
I don't want to set it as PK, cause that value will be constantly changing.
What I mean by that:
To allow a flow, that we are able to reschedule execution of another webhooks batch we need new deduplication id for AWS SQS FIFO Queue, cause there is 5 minutes time frame when no new tasks will be proceeded. So to be able to instantly put new task in queue we need new brand uuid that all other workers/processes will be trying to use when executing a task.